pygmt/clib/session.py

```diff
@@ -1783,9 +1788,13 @@ def virtualfile_in(  # noqa: PLR0912
         mincols
             Number of minimum required columns. Default is 2 (i.e. require x and y
             columns).
-        required_data : bool
-            Set to True when 'data' is required, or False when dealing with
-            optional virtual files. [Default is True].
+        required : bool
+            Set to True when 'data' is required, or False when dealing with optional
+            virtual files. Default is True.
```

Comment:
Based on logic in `_validate_data_input`. Feel free to reword as needed.

```suggestion
            Set to True when 'data' or (`x` and `y`) is required . Set to False when
            dealing with optional virtual files. Default is True.
```